diff --git a/tests/integration/rdb.tcl b/tests/integration/rdb.tcl
index da9608e0b..88bb98d53 100644
--- a/tests/integration/rdb.tcl
+++ b/tests/integration/rdb.tcl
@@ -113,7 +113,7 @@ set server_path [tmpdir "server.rdb-startup-test"]
 start_server [list overrides [list "dir" $server_path] keep_persistence true] {
     test {Server started empty with non-existing RDB file} {
         debug_digest
-    } {0000000000000000000000000000000000000000}
+    } {0000000000000000000000000000000000000000} {needs:debug}
     # Save an RDB file, needed for the next test.
     r save
 }
@@ -121,7 +121,7 @@ start_server [list overrides [list "dir" $server_path] keep_persistence true] {
 start_server [list overrides [list "dir" $server_path] keep_persistence true] {
     test {Server started empty with empty RDB file} {
         debug_digest
-    } {0000000000000000000000000000000000000000}
+    } {0000000000000000000000000000000000000000} {needs:debug}
 }
